ocaml/xapi/export_raw_vdi.ml

```diff
@@ -53,6 +53,12 @@ let localhost_handler rpc session_id vdi (req: Http.Request.t) (s: Unix.file_des
              Http_svr.headers s headers;
              match format with
              | Raw | Vhd ->
+               if format = Vhd then begin
+                 let size = Db.VDI.get_virtual_size ~__context ~self:vdi in
+                 if size > 2199023255552L
```

Comment:
Our supported limit is 2TiB-4GiB, see https://docs.citrix.com/en-us/xenserver/current-release/system-requirements/configuration-limits.html, I think we should use that.
I think this constant is already defined here: https://github.com/xapi-project/xen-api/blob/680e7b1feed1186e998a845fe4979bc9dfc9cd4a/ocaml/quicktest/quicktest_max_vdi_size.ml#L6

The constant itself should probably be moved to `constants.ml`, and both this code and the unit test could refer to it.

I think @gaborigloi tested various values in a PR a while ago and we determined that TB means TiB in those limits. (from our official docs it is not clear whether TB=10^12 bytes, or TB=TiB=2^40 bytes